packages/blockprotocol/core.d.ts

```diff
@@ -37,6 +45,13 @@ export type BlockProtocolCreateEntitiesAction<T> = {
   entityTypeVersionId?: string | null;
   data: T;
   accountId?: string | null;
+  links?: DistributedOmit<
+    BlockProtocolCreateLinksAction,
+    | "sourceAccountId"
```

Comment:
Does this need renaming to `sourceEntityAccountId`?

Reply:
Same for all the other instances of the string.

Reply:
Nope - see the PR description